community/kernel/src/main/java/org/neo4j/kernel/impl/factory/GraphDatabaseFacadeFactory.java

```diff
@@ -90,6 +90,7 @@
 
         // Kept here to have it not be publicly documented.
         public static final Setting<String> lock_manager = setting( "lock_manager", Settings.STRING, "" );
+        public static final Setting<String> record_format = setting( "record_format", Settings.STRING, "" );
```

Comment:
If the high limit format is available it will choose it unless you specify "community". The empty string as well as "highlimit" will result in using the high limits if it is available.